src/wp-includes/plugin.php

```diff
@@ -590,7 +590,7 @@ function remove_all_actions($tag, $priority = false) {
  *
  * @param string $tag         The name of the filter hook.
  * @param array  $args        Array of additional function arguments to be passed to apply_filters().
- * @param string $version     The version of WordPress that deprecated the hook.
+ * @param string $version     The version of ClassicPress that deprecated the hook.
```

Comment:
Let's say "The version of ClassicPress or WordPress" for now.  We'll prefix these version numbers with `WP-` if they are WordPress versions.  See #27.